# Pointswithnormal.msg
Header header
float32[] points_x
float32[] points_y
float32[] normals_x
float32[] normals_y
